scripts/wise/check-fees.ts

```diff
@@ -256,6 +259,114 @@ program.command('fix [hosts]').action(async hosts => {
   sequelize.close();
 });
 
+type RefundData = {
+  [MIGRATION_DATA_KEY]: {
+    paymentOption: QuoteV3PaymentOption;
+    quote: QuoteV3;
+  };
+  [POST_MIGRATION_DATA_KEY]: {
+    paymentOption: QuoteV3PaymentOption;
+    quote: QuoteV3;
+  };
+};
+
+program.command('refund [hosts]').action(async hosts => {
+  hosts = hosts ? hosts.split(',') : [];
+  console.log(`Refunding Collectives for fees on expenses previously checked for hosts ${hosts.join(', ')}...`);
+  for (const slug of hosts) {
+    const expenses = await models.Expense.findAll({
+      where: {
+        status: 'PAID',
+        data: {
+          [MIGRATION_DATA_KEY]: { [Op.ne]: null },
+          [POST_MIGRATION_DATA_KEY]: { [Op.ne]: null },
+          [REFUNDED_MIGRATION_DATA_KEY]: null,
+        },
+      },
+      include: [
+        { model: models.Collective, as: 'host', required: true, where: { slug } },
+        { model: models.Collective, as: 'collective' },
+      ],
+    });
+
+    const groupedExpenses = groupBy(expenses, 'CollectiveId');
+    let totalRefunded = 0;
+    for (const key in groupedExpenses) {
+      const collectiveExpenses = groupedExpenses[key];
+      const collective = collectiveExpenses[0].collective;
+      const host = collectiveExpenses[0].host;
+      let currency;
+      try {
+        const totalDiff = collectiveExpenses.reduce((acc, expense) => {
+          const data = expense.data as RefundData;
+          const beforePaymentOption = data[POST_MIGRATION_DATA_KEY].paymentOption;
+          const afterPaymentOption = data[MIGRATION_DATA_KEY].paymentOption;
+          assert.equal(beforePaymentOption.sourceCurrency, afterPaymentOption.sourceCurrency);
+          assert.equal(beforePaymentOption.sourceCurrency, expense.host.currency);
+          currency = beforePaymentOption.sourceCurrency;
+          const diff = beforePaymentOption.sourceAmount - afterPaymentOption.sourceAmount;
+          return round(acc + diff, 2);
+        }, 0);
+        if (totalDiff < 0) {
+          console.log(`\tRefunding ${collective.slug} for a total of ${totalDiff} ${currency}`);
+          const amountInHostCurrency = Math.abs(totalDiff * 100);
+          totalRefunded += amountInHostCurrency;
+          const hostCurrencyFxRate = await getFxRate(collective.currency, host.currency);
+          const amount = Math.round(amountInHostCurrency / hostCurrencyFxRate);
+          const transactionData = {
+            type: TransactionTypes.CREDIT,
+            kind: TransactionKind.PAYMENT_PROCESSOR_COVER,
```

Comment:
Maybe ADDED_FUNDS or CONTRIBUTION would be better.

It's not expected to have  PAYMENT_PROCESSOR_COVER not part of a TransactionGroup and this can create issues and break some consistency checks.